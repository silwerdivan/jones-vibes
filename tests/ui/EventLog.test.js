import { screen, getByTestId } from '@testing-library/dom';
import '@testing-library/jest-dom';
import { JSDOM } from 'jsdom';
import fs from 'fs';
import path from 'path';

const html = fs.readFileSync(path.resolve(__dirname, '../../index.html'), 'utf8');

describe('Event Log Snapshot', () => {
    beforeEach(() => {
        document.body.innerHTML = html;
    });

    test('should match the snapshot for the event log', () => {
        const eventLog = screen.getByTestId('game-log');
        expect(eventLog).toMatchSnapshot();
    });
});
