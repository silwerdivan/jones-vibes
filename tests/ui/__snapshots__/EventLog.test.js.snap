// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Event Log Snapshot should match the snapshot for the event log 1`] = `
<div
  class="event-log"
  data-testid="game-log"
>
  
                
  <h3>
    Game Log
  </h3>
  
                
  <div
    class="log-content"
  />
  
            
</div>
`;
